.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "backup"]
```

Comment:
Not crazy about creating this hard dependency. For one thing promtimer is public and backup is private. But also it the dependency seems unnecessary. I presume the only actual dependency is on cbmstatparser. I think this can just be expressed in the docs - that one needs to have cbmstatparser available on the path when running against a backup archive. Suggest remove the submodule.